pkg/app/middleware.go

```diff
@@ -61,3 +62,14 @@ func SetupCORSMiddleware(handler http.Handler, domains []string) http.Handler {
 	})
 	return corsHandler.Handler(handler)
 }
+
+func DisableV1Middleware(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if strings.Contains(r.URL.Path, "/assisted-install/v1/") {
+			w.WriteHeader(http.StatusNotFound)
+			w.Write([]byte("v1 API is currently disabled"))
```

Comment:
Lint is failing here but I wouldn't do anything differently if this write fails.
Is it worth passing in a logger just so I can log that we failed to write this message?

Reply:
I think that something like this will fix the lint error.
Besides log there is nothing we really can do. 
```suggestion
			_, _ = w.Write([]byte("v1 API is currently disabled"))
```